src/matrixrtc/MatrixRTCSession.ts

```diff
@@ -786,10 +846,28 @@ export class MatrixRTCSession extends TypedEventEmitter<
     /**
      * Call this when the Matrix room members have changed.
      */
-    public onRoomMemberUpdate = (): void => {
+    private readonly onRoomMemberUpdate = (): void => {
         this.recalculateSessionMembers();
     };
 
+    /**
+     * Call this when a sticky event update has occured.
+     */
+    private readonly onStickyEventUpdate: RoomStickyEventsMap[RoomStickyEventsEvent.Update] = (
+        added,
+        updated,
+        removed,
+    ): void => {
+        this.logger.debug("Sticky event update", { added, updated, removed });
```

Comment:
We shouldn't log these as it would leak sensitive event contents in rageshakes and is also quite a lot of data to serialize.